parsing.py

```diff
@@ -263,6 +266,8 @@ def _parse_ms2lda(usi: str) -> Tuple[sus.MsmsSpectrum, str]:
 
 # Parse MSV or PXD library.
 def _parse_msv_pxd(usi: str) -> Tuple[sus.MsmsSpectrum, str]:
+    if usi.startswith('mzdraft:'):
```

Comment:
`mzdraft` should normally never happen here because it is processing in parsing_legacy.py.

We've also maybe had this long enough. Rather than introducing potential technical debt, maybe we should just remove `mzdraft` support and error on those USIs so that people are encouraged to fix them (if anyone still uses `mzdraft` even).

Reply:
I was getting errors without this in the testing. According to @mwang87 all GNPS identifiers should be able to be passed with either `mzdraft` or `mzspec`, and this is how the unit testing is setup.

Reply:
Weird. [Normally `mzdraft` USIs should be parsed by the legacy code](https://github.com/mwang87/MetabolomicsSpectrumResolver/blob/master/parsing.py#L89). This PR didn't change that, but maybe not a bad time to open this up for a broader discussion to see whether it makes sense to still support this legacy format? @mwang87 What do you think?